Dispatcher/Views/Auth/SignupOrLoginView.swift

```diff
@@ -0,0 +1,289 @@
+//
```

Comment:
What's the idea behind creating this dedicated view and not just placing the text fields directly in the Auth screen?